cmd/crc/cmd/root.go

```diff
@@ -33,21 +33,28 @@ var rootCmd = &cobra.Command{
 	},
 }
 
-var globalForce bool
+var (
+	globalForce bool
+	viper       *crcConfig.ViperStorage
+	config      *crcConfig.Config
+)
 
 func init() {
 	if err := constants.EnsureBaseDirExists(); err != nil {
 		logging.Fatal(err.Error())
 	}
-	if err := config.InitViper(); err != nil {
+	var err error
+	viper, err = crcConfig.NewViperStorage(constants.ConfigPath, constants.CrcEnvPrefix)
+	if err != nil {
 		logging.Fatal(err.Error())
 	}
-	cmdConfig.RegisterSettings()
+	config = crcConfig.New(viper)
+	cmdConfig.RegisterSettings(config)
```

Comment:
Seeing config commands being passed explicitly their config object (I understand it's because they are in a `config` subdir/subpackage), and setup/start/... making use of a package global version of it feels a bit unfair :)

Reply:
Yes we will improve that!